README.md

```diff
@@ -144,6 +144,8 @@ ENDINPUT
 BEGININSTRUCTION Was ist 1+1? ENDINSTRUCTION ASSISTANT:
 ```
 
+*Note: Unfortunately, a typo has sneaked in during training ("ENDINCSTRUCTION"), which will be corrected in the next model version. For optimal performance, the typo should be adopted in the prompt.*
```

Comment:
could you add that the typo was only in the System Message / Instruction (and not in the actual prompt format)? Thanks :-)

Reply:
Of course, I hope it's clear now!
By now it's probably a good idea to squash the commits when merging :)